packages/squiggle-lang/src/rescript/utility/XYShape.res

```diff
@@ -1,4 +1,18 @@
-open PointSetTypes
+type xyShape = {
```

Comment:
can we go ahead and capitalize `Utility` while we're here?

Reply:
Good point, done